### PR TITLE
Add multi-site topic

### PR DESCRIPTION
**Pull request by dobooth**
## Description

This PR adds a topic for multi-site setup for PWA/UPWARD

## Related Issue

https://jira.corp.magento.com/browse/PWA-2562

### CODE REVIEW

**Review by tjwiebell — CHANGES_REQUESTED** [review]
Good starting point. Once theming work is closer to landing we can expand the use cases to include that scenario.

I'm not sure about the location, since this topic is specific to upward-php, and really only used in cloud environments. Thinking it should colocate closer to the [Deploy to Adobe Commerce](https://developer.adobe.com/commerce/pwa-studio/tutorials/production-deployment/adobe-commerce/) tutorial.

**Review by tjwiebell — APPROVED** [review]

**Review by bdenham — APPROVED** [review]
So I went through this as the newbie I am—but unfortunately, I don't have a running instance at the moment to test and walk through things as I would normally do. 

@tjwiebell : Can you make sure my changes are still correct? I made some assumptions that I'm not yet sure about. Thanks!

Approved pending @tjwiebell's review and any requested updates.

File: src/pages/guides/packages/upward/multi-site.md
Comment (bdenham) [reply src/pages/guides/packages/upward/multi-site.md:57]: @tjwiebell can you verify the command above?

**Pull request merged**
